src/Event/UpdateEvent.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex\Event;
+
+use Composer\Script\Event;
+
+class UpdateEvent extends Event
+{
+    private static $defaultParameters = [
```

Comment:
I'd suggest setting the default value directly in the property bellow and getting rid of this, later, in constructor:
```php
$this->parameters = $parameters + $this->parameters;
```

Reply:
The array has been removed in favor of a boolean flag :)